samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -130,6 +130,54 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Low Power Node Support
+======================
+
+The mesh light switch sample can also be run as a Low Power Node (LPN). To make the light switch run as a LPN, build the sample using the following command:
+
+  .. code-block:: console
+
+     west build -b <BOARD> -p -- -DOVERLAY_CONFIG="lpn.conf"
+
+The :file:`lpn.conf` is a configuration overlay that will enable the LPN feature, as well as altering certain configuration options to further lower the power consumption. To review the specific alterations, please look at the :file:`lpn.conf`.
+
+While running the sample with the LPN configuration, the fourth Gen. OnOff Client instance will be omitted and the corresponding button will instead be used to temporarily enable GATT proxy on the LPN device. Running GATT proxy continuously will consume considerable power, and therefore the application is set to automatically disable GATT proxy advertisement after a period of 30 seconds, given that the device is provisioned. In this way the user will be able to manually re-enable GATT proxy for a short period by pressing button 4 on the device, and can then connect over GATT and perform necessary configuration. When the configuration is complete the user will then disconnect from the LPN and the GATT proxy advertisement will stop.
+
+Friendship establishment will happen automatically after provisioning the LPN light switch, given that a :ref:`bluetooth_mesh_light` sample is running and is provisioned into the same mesh network.
+
+The following table shows a list of the supported boards for the LPN configuration, the average power consumption running as a standard node, and the average power consumption running as a LPN node:
+
+.. table::
+   :align: center
+
+   ===================  =========================  ====================
+   Board                Avg. Consumption Standard  Avg. Consumption LPN
+   ===================  =========================  ====================
+   nrf52dk_nrf52832     7.14 mA                    13.69 uA
+   nrf52840dk_nrf52840  6.71 mA                    14.63 uA
+   nrf52833dk_nrf52833  6.10 mA                    14.43 uA
```

Comment:
What is the sample rate used here? You will get an artificially high value of average current if the sample rate is too small. Ideally, the sample rate of PPK should be set to as high as possible for 1-minute window measurement.

Reply:
I belive I used the second highest rate to allow a longer sampling period. I tried the highest for a short while and got simmilar results.